modules/chre/Kconfig

```
@@ -0,0 +1,70 @@
+# Copyright (c) 2021 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config ZEPHYR_CHRE_MODULE
+	bool
+
+menuconfig CHRE
+	bool "CHRE Support"
+	help
+	  This option enables the CHRE library.
+
+if CHRE
+
+module = CHRE
+module-str = chre
+source "subsys/logging/Kconfig.template.log_config"
+
+config CHRE_MESSAGE_TO_HOST_MAX_SIZE
+	int "Maximum message size to send to the host (bytes)"
+	default 4000
+	help
+	  Defines the maximum message size that can be sent to the host.
+
+config CHRE_ASSERTIONS
+	bool "Enable CHRE assertions"
+	help
+	  When enabled, CHRE will include various assertions which will trigger
+	  calls to chreAbort if resolve to false.
+
+config CHRE_AUDIO_SUPPORT_ENABLED
```

```suggestion
config CHRE_AUDIO_SUPPORT
```

I would prefer to remove the `_ENABLED` postfix for all these Kconfig options.

---

Please comment on https://android-review.googlesource.com/c/platform/system/chre/+/1952616